### ISSUE TITLE
How to Evaluate the model?

### ISSUE THREAD

**Issue by Deepanshu-beep** [kmaninis/OSVOS-PyTorch#39]
@kmaninis How to get the Evaluation metrics as mentioned in the paper. I've got the pth model after training the model from the code but don't know how to evaluate it. I'm new to research domain so I'd be thankful if you could guide me through it.

Also, can we get a combined model for all object masks detection or would it be just individual pth models?

**Comment by kmaninis** [comment]
hi, please use [this code](https://github.com/davisvideochallenge/davis-matlab/tree/davis-2016) on DAVIS 2016. You can find more info in the [DAVIS main page](https://davischallenge.org/davis2016/soa_compare.html).

As for the output, you get an individual pth model per object.
